src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -114,21 +115,26 @@
 
 	private static final String FIND_GERMPLASM_BY_GIDS = " g.gid in (:gids) ";
 
+	private static final String FIND_GERMPLASM_BY_MGID = " g.mgid = :gid AND  g.deleted = 0  and g.grplce = 0 ORDER BY g.gid";
```

Comment:
Better to just append these clauses in the query where they are used rather than extracting them as constants

Reply:
Done